src/components/component-playground.jsx

```diff
@@ -180,15 +179,41 @@ module.exports = React.createClass({
     </ul>;
   },
 
+  _renderPreview: function() {
+    return <div ref="previewContainer" className={this._getPreviewClasses()}>
+      {this.loadChild('preview')}
+    </div>
+  },
+
   _renderContentFrame: function() {
     return <div ref="contentFrame" className={this._getContentFrameClasses()}>
-      <div ref="previewContainer" className={this._getPreviewClasses()}>
-        {this.loadChild('preview')}
-      </div>
-      {this.props.editor ? this._renderFixtureEditor() : null}
+      {this.props.editor ? this._renderSplitPane() : this._renderPreview()}
     </div>
   },
 
+  _renderSplitPane: function() {
+    return <SplitPane
+      split={this._getOrientationDirection()}
+      ref='editorPreviewSplitPane'
+      defaultSize={localStorage.getItem('splitPos')}
+      onChange={size => localStorage.setItem('splitPos', size)}
+      minSize={20}
```

Comment:
@bogdan-ciobanu-hs Isn't this the minSize and not the default one? I don't want to go back to the humongous edit panel. Whats wrong with the min size if you can adjust the panel as you want?